composer.json

```
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0"
+        "php": "^5.6|^7.0|^8.0"
```

```suggestion
        "php": "^5.6 || ^7.0 || ^8.0"
```
Single pipe works but double pipe seems more appropriate.
See https://github.com/composer/composer/issues/6755#issuecomment-337588505